examples/fit_ESR_voigt.py

```diff
@@ -0,0 +1,137 @@
+from pint import UnitRegistry
+from pyspecdata import *
+from pyspecProcScripts import *
+from pyspecProcScripts import QESR_scalefactor
+from sympy import symbols, Symbol, latex
+from sympy import exp as s_exp
+import numpy as np
+
+logger = init_logging(level="debug")
+fieldaxis = "$B_0$"
+exp_type = "francklab_esr/alex"
+with figlist_var() as fl:
+    for filenum, (thisfile, calibration, diameter, background) in enumerate(
+        [
+            (
+                "230418_200uM_TEMPOL_3_redo2.DSC",
+                "230202",
+                "QESR caps",
+                find_file("230418_water.DSC", exp_type=exp_type)["harmonic", 0],
+            ),
+        ]
+    ):
+        # {{{ load data in and rescale and center about 0
+        d = find_file(thisfile, exp_type=exp_type)
+        if "harmonic" in d.dimlabels:
+            d = d["harmonic", 0]
+        color = d.get_plot_color()
+        d /= QESR_scalefactor(d, calibration_name=calibration, diameter_name=diameter)
+        center_field = d.getaxis(fieldaxis)[r_[0, -1]].mean()
+        d -= d[fieldaxis, -100:].data.mean()
+        d.setaxis(fieldaxis, lambda x: x - center_field)
+        background = ndshape(d).alloc()
+        background.copy_axes(d)
+        background = background.interp(fieldaxis, d.getaxis(fieldaxis))
+        d -= background
+        # }}}
+        d.ift(fieldaxis, shift=True)
+        d *= 2
+        d[fieldaxis:0] *= 0.5
```

Comment:
why do this? this is only relevant when you are converting from symmetric to causal signal.